src/app/book/[bookId]/page.tsx

```diff
@@ -24,6 +25,7 @@ const BookDetailPage = () => {
   return (
     <div>
       <BookDetailContainer book={validBook} />
+      <BookDetailContainer2 book={validBook} />
```

Comment:
what is the difference between BookDetailContainer and BookDetailContainer2?
Could BookDetailContainer2 be renamed something different?